### PR TITLE
Add appstore url to mobile apps.

### PR DESCRIPTION
**Pull request by jeremyfaller**
## Proposed Changes

* Add appstore links to the mobile apps, stored in the DB.

**Release Note**

```release-note
Update the Mobile Apps to store links to their appstore.
```

### CODE REVIEW

File: cmd/server/assets/mobileapps/show.html [hunk 1]
```diff
@@ -30,6 +30,9 @@ <h1>{{$app.Name}}</h1>
           <dt>App name</dt>
           <dd>{{$app.Name}}</dd>
 
+          <dt>AppStore link</dt>
+		  <dd><a href="{{$app.URL}}" target="_blank">{{$app.URL}}</a></dd>
```
Comment (sethvargo) [inline cmd/server/assets/mobileapps/show.html:34]:
```suggestion
          <dd><a href="{{$app.URL}}" target="_blank">{{$app.URL}}</a></dd>
```

File: pkg/database/mobile_app.go [hunk 2]
```diff
@@ -57,6 +57,9 @@ type MobileApp struct {
 	// RealmID is the id of the mobile app.
 	RealmID uint `gorm:"column:realm_id;"`
 
+	// URL is the link to the app in it's appstore.
+	URL string `gorm:"type:varchar(2048);not null;default:''"`
```
Comment (sethvargo) [inline pkg/database/mobile_app.go:61]:
```suggestion
	URL string `gorm:"type:text; not null; default:''"`
```

File: pkg/database/migrations.go [hunk 3]
```diff
@@ -1427,6 +1427,30 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
...
+		{
+			ID: "00058-AddAppStoreLink",
+			Migrate: func(tx *gorm.DB) error {
+				if false {
+					return nil
+				}
+				sqls := []string{
+					"ALTER TABLE mobile_apps ADD COLUMN IF NOT EXISTS url VARCHAR(2048) NOT NULL",
+					"ALTER TABLE mobile_apps ALTER COLUMN url SET DEFAULT ''",
+					"UPDATE mobile_apps SET url='' WHERE url=NULL",
```
Comment (sethvargo) [inline pkg/database/migrations.go:1439]: Should we just allow this column to be nullable instead?

File: pkg/database/migrations.go [hunk 4]
```diff
@@ -1427,6 +1427,30 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
...
 			},
 		},
+		{
+			ID: "00058-AddAppStoreLink",
+			Migrate: func(tx *gorm.DB) error {
+				if false {
+					return nil
+				}
+				sqls := []string{
+					"ALTER TABLE mobile_apps ADD COLUMN IF NOT EXISTS url VARCHAR(2048) NOT NULL",
```
Comment (sethvargo) [inline pkg/database/migrations.go:1437]: Use `text` type

**Review by flagxor — COMMENTED** [review]
lgtm

**Comment by jeremyfaller** [comment]
PTAL, @sethvargo

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**